adobe/react-spectrum
# Add conditional DnD support to Table component

Closes https://github.com/adobe/react-spectrum/issues/5609

## ✅ Pull Request Checklist:


## 📝 Test Instructions:


1. Go to http://localhost:9003/?path=/story/react-aria-components--dnd-table-example&args=isDisabledFirstTable:true;isDisabledSecondTable:true&providerSwitcher-express=false&strict=true
2. Open the Controls tab in Storybook and set isDisabledFirstTable to true.
3. Verify that the table on the left side cannot perform drag and drop actions.

## 🧢 Your Project:
